src/redux/reducers/trendsStateByUser.js

```diff
@@ -20,13 +20,15 @@ import update from 'react-addons-update';
 
 import * as actionTypes from '../constants/actionTypes';
 
-const FOCUSED_SLICE = 'focusedCbgSlice';
-const FOCUSED_KEYS = 'focusedCbgSliceKeys';
+const FOCUSED_CBG_SLICE = 'focusedCbgSlice';
+const FOCUSED_CBG_KEYS = 'focusedCbgSliceKeys';
+const FOCUSED_SMBG_AVG = 'focusedSmbgAvg';
```

Comment:
Can we do `FOCUSED_SMBG_RANGE_AVG` and `focusedSmbgRangeAvg` like elsewhere, for consistency?